### PR TITLE
HomeMatic README for #726

### PR DESCRIPTION
**Pull request by TravelinMax**
For #726

### CODE REVIEW

File: homematic/README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "HomeMatic CCU" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+Currently, all .
```
Comment (frenck) [inline homematic/README.md]: all... ?

File: homematic/README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Toggle the "Show in sidebar" option, which adds the Configurator to the main menu.
+2. Start the add-on.
+3. Refresh your browser, the "Configurator" is now visible in the sidebar.
+4. Click on the "Configurator" menu option and start configuring!
```
Comment (frenck) [inline homematic/README.md]:
```suggestion
4. Click on the "Configurator" menu option and start configuring it!
```

File: homematic/README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+By default, it hides the `__pycache__` folders.
+
+### Option: `ssh_keys` (required)
+
+A list of filenames containing SSH private keys. These can be used to allow for access to remote git repositories.
+
+## Known issues and limitations
+
+- This add-on is, by default, configured for use with Hass.io Ingress. If you
+  wish to access the add-on via a its own port directly, you can simply
```
Comment (frenck) [inline homematic/README.md]:
```suggestion
  wish to access the add-on via its own port directly, you can simply
```

**Comment by TravelinMax** [comment]
I have some more changes I need to make (and some I just need to push), I've just been really really busy 😭. I'll finish them this week I promise!

**Comment by TravelinMax** [comment]
@frenck this one is ready for review 😁

**Pull request merged**